cfy_manager/constants.py

```diff
@@ -25,6 +25,9 @@
 CLOUDIFY_SUDOERS_FILE = join(SUDOERS_INCLUDE_DIR, CLOUDIFY_USER)
 INITIAL_INSTALL_FILE = join(CLOUDIFY_HOME_DIR, '.install')
 
+COMMON_RUN_DIR = '/var/run/cloudify'
+# Must be identical to COMMON_LOCK_DIR in cloudify-premium/ha/utils.py
```

Comment:
do you have a pr for this as well?

Reply:
Yes: https://github.com/cloudify-cosmo/cloudify-premium/pull/453